lib/usb-hid-msc-lib.robot

```diff
@@ -33,3 +33,24 @@ Download ISO And Mount As USB
             Skip    unsupported
         END
     END
+
+Copy ISO And Mount As USB
+    [Documentation]    Mounts the desired ISO as USB stick,
+    ...    either via PiKVM or Qemu
+    [Arguments]    ${img_path}    ${img_file}
+
+    ${img_dir}    ${img_name}=    Split Path    ${img_path}
+
+    Copy File    ${img_file}    ${CURDIR}/../dl-cache/${img_name}
+
+    IF    "${MANUFACTURER}" == "QEMU"
+        Remove Drive From Qemu
+        Add USB To Qemu    img_name=${img_path}
+    ELSE
+        IF    "${DUT_CONNECTION_METHOD}" == "pikvm"
+            Copy Image To PiKVM    ${PIKVM_IP}    ${img_file}    ${img_name}
```

Comment:
In `Download ISO And Mount As USB` a different command was used: `Upload Image To PiKVM ` what is the reason for change?

I cant find this keyword. Did you ad it on your own? Could you add it to the PR?

Reply:
Could you point to that KWD?

Reply:
I left some suggestions regarding this kwd in pikvm module here: https://github.com/3mdeb/pikvm-rest-api/pull/16/files#r1812874751